extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -101,7 +102,7 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
 
     public identifier: string = AnalyticsPluginIdentifier; // do not change name or priority
     public priority: number = 180; // take from reserved priority range 100- 200
-    public readonly config: IConfig & IConfiguration;
+    public readonly config: IAnalyticsConfig;
```

Comment:
This exposed public config MUST still be IConfig & IConfiguration, its only the _extConfig that should be the IAnalyticConfig and all config usages within this class should be using the _extConfig